src/Parser/Warrior/Arms/Modules/BuffDebuff/ColossusSmashUptime.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class ColossusSmashUptime extends Module {
+  static dependencies = {
+    enemies: Enemies,
+  };
+
+  suggestions(when) {
+    const colossusSmashUptime = this.enemies.getBuffUptime(SPELLS.COLOSSUS_SMASH_DEBUFF.id) / this.owner.fightDuration;
+
+    when(colossusSmashUptime).isLessThan(0.95)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.COLOSSUS_SMASH_DEBUFF.id} /> uptime can be improved. Try to pay more attention to your debuff on the Boss</span>)
```

Comment:
Can you expand on just why it is important to keep the debuff up? Does is provide increased damage / used for additional procs / etc?